tss-esapi/tests/integration_tests/abstraction_tests/public_tests.rs

```diff
@@ -70,44 +69,38 @@ mod public_rsa_test {
     #[test]
     fn test_public_to_decoded_key_rsa() {
         let public_rsa = get_ext_rsa_pub();
-        let default_exponent = IntegerAsn1::from_bytes_be_signed(65537_u32.to_be_bytes().to_vec());
-        let decoded_key = DecodedKey::try_from(public_rsa)
+        let default_exponent = BigUint::from(65537_u32);
+        let key = rsa::RsaPublicKey::try_from(&public_rsa)
             .expect("Failed to convert Public structure to DecodedKey (RSA).");
-        match decoded_key {
-            DecodedKey::RsaPublicKey(key) => {
-                assert_eq!(
-                    key.public_exponent, default_exponent,
-                    "RSA exponents are not equal."
-                );
-                assert_eq!(key.modulus.as_unsigned_bytes_be(), RSA_KEY);
-            }
-            DecodedKey::EcPoint(..) => panic!("RSA key was decoded to EcPoint!"),
-        }
+        assert_eq!(key.e(), &default_exponent, "RSA exponents are not equal.");
+        assert_eq!(key.n().to_bytes_be(), RSA_KEY);
     }
 
     #[test]
     fn test_public_to_subject_public_key_info_rsa() {
         let public_rsa = get_ext_rsa_pub();
-        let default_exponent = IntegerAsn1::from_bytes_be_signed(65537_u32.to_be_bytes().to_vec());
-        let key = SubjectPublicKeyInfo::try_from(public_rsa)
+        let key = SubjectPublicKeyInfoOwned::try_from(&public_rsa)
             .expect("Failed to convert Public structure to SubjectPublicKeyInfo (RSA).");
-        assert_eq!(key.algorithm, AlgorithmIdentifier::new_rsa_encryption());
-        match key.subject_public_key {
-            PublicKey::Rsa(key) => {
-                assert_eq!(key.public_exponent, default_exponent);
-                assert_eq!(key.modulus.as_unsigned_bytes_be(), RSA_KEY)
-            }
-            _ => panic!("PublicKey of SubjectPublicKeyInfo is not an instance for RSA"),
-        }
+        let default_exponent = BigUint::from(65537_u32);
```

Comment:
Same thing here try to use a constant instead.

Reply:
I don't want to expose the constant in the API, I will redefine it.